README.md

```diff
@@ -20,14 +20,34 @@ npm install react-transition-group --save
 yarn add react-transition-group
 ```
 
+## CDN / External
+
+To exclude the react-transition-group package from your primary build and use an external CDN when bundling, link
+to the following CDN:
+
+[https://unpkg.com/react-transition-group/dist/react-transition-group.min.js](https://unpkg.com/react-transition-group/dist/react-transition-group.min.js)
+
```

Comment:
Various libraries exclude this package but depend on it. I'd argue that library components absolutely should be used from a CDN as the alternative reinforces bundling huge apps, SPA, large bundles with users not sure how to split/reuse common framework components. E.g. React, Bootstrap, etc. This is an extension of React essentially. 

Nonetheless, would you prefer I move to the bottom or remove?